data/WAH_2022_1/data.csv

```diff
@@ -170,28 +170,28 @@ Centaurea,Centaurea melitensis, general biology. reproduction. reproduces by see
 Chamelaucium,Chamelaucium uncinatum, general biology. growth form. shrub. reproduction. seed. dispersal. ,perennial,inferred_from_growth_form,,,shrub,shrub,,,,,,,woody,
 Chasmanthe,Chasmanthe floribunda," general biology. growth form. geophyte. life form. annually renewed corm. reproduction. primarlily seed, also corm offsets. dispersal. ",perennial,inferred_from_growth_form,,,geophyte,geophyte,,,,,,,woody,inferred_from_growth_form
 Chenopodium,Chenopodium album," general biology. growth form. herb. life form. annual. reproduction. seed. dispersal. infestations may be very dense and compete vigorously with other plants. manual removal, particularly when plants are young, is effective due to shallow root systems, and can be done up to stem elongation and before seeding in summer. relatively tolerant to many herbicides and any sub-lethal rates of herbicide will result in the same high seed germination rates as unsprayed plants. read the manufacturers' labels and material safety data sheets before using herbicides. for further information consult the australian pesticides and veterinary medicines authority to determine the status of permits for your situation or state. ",annual,annual,,,herb,herb,,,,,dense,dense,herbaceous,
 Chenopodium,Chenopodium ambrosioides, general biology. growth form. herb. life form. perennial. reproduction. seed. dispersal. read the manufacturers' labels and material safety data sheets before using herbicides. for further information consult the australian pesticides and veterinary medicines authority to determine the status of permits for your situation or state. ,perennial,perennial,,,herb,herb,,,,,,,herbaceous,
-Chordifex,Chordifex reseminans," rhizomes tufted, ",,,,,graminoid,,,,dioecious,,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex gracilior," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
+Chordifex,Chordifex reseminans," rhizomes tufted, ",,,,,graminoid,,,,,,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
+Chordifex,Chordifex gracilior," rhizomes spreading, ",,,,,graminoid,,,,,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
 Chordifex,Chordifex amblycoleus," rhizomes tufted, glabrous. dioecious (with separate male and female plants). ",,,,,graminoid,,,,dioecious,dioecious,rhizomatous,rhizomes,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex chaunocoleus," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
-Chordifex,Chordifex isomorphus," rhizomes spreading, ",,,,,graminoid,,,,dioecious,,rhizomatous spreading,rhizomes spreading,herbaceous,inferred_from_growth_form
```

Comment:
Why was this removed

Reply:
Still investigating this one, I'd made the changes a long time ago so want to check the raw data one last time

Reply:
Finished triple checking all the changes and I'm satisfied that they are all intentional and correct ;)